src/main/scala/com/github/saurfang/sas/mapred/SasRecordReader.scala

```diff
@@ -29,9 +29,11 @@ class SasRecordReader() extends RecordReader[NullWritable, Array[Object]] with L
   private var recordCount: Int = 0
   private var lastPageBlockCounter: Int = 0
   private var recordValue: Array[Object] = null
+  
+  val log = LogManager.getRootLogger
```

Comment:
Can you use the standard logger by classname instead of root logger? i.e. something like `val logger = Logger.getLogger(this.getClass.getName)`